source/models/defining-models.md

```diff
@@ -78,9 +78,27 @@ export default DS.Model.extend({
 ```
 
 Ember Data supports attribute types of `string`, `number`, `boolean`,
-and `date`. Which coerce the value to the JavaScript type that matches
+and `date`, which coerce the value to the JavaScript type that matches
 its name.
 
+```app/models/person.js
+export default DS.Model.extend({
+  name: DS.attr('string'),
+  age: DS.attr('number'),
+  admin: DS.attr('boolean'),
+  birthday: DS.attr('date')
+});
+```
+
+The `string` and `number` transforms simply use the `String` and `Number`
+functions respectively to coerce the attributes.
+
+The `boolean` transform can handle values other than `true` or
```

Comment:
The boolean conditions being split in two sentences is confusing. First all the truthy values, then all the falsey values.